fix: Reduce number of warnings on XSUAA selection (#1593)
SAP/cloud-sdk-js: CHANGELOG.md

## Improvements

> - [xsuaa] Reduce the number of warn message when selecting a XSUAA instance.

Reviewer: ```suggestion
- [core] Reduce the number of warnings when selecting an XSUAA instance.
```
